docs/bin/plugin_formatter.py

```diff
@@ -91,7 +91,7 @@ def rst_ify(text):
     try:
         t = _ITALIC.sub(r"*\1*", text)
         t = _BOLD.sub(r"**\1**", t)
-        t = _MODULE.sub(r":ref:`\1 <\1>`", t)
+        t = _MODULE.sub(r":ref:`\1 <\1_module>`", t)
```

Comment:
This will work for what we currently have, though we will not be able to link to a non-module plugin using using the`  M(..)` option. I think that's OK though.

Reply:
Yeah, for the future, we'll have to have macros for each of the plugin types.  This simple regex can't handle that.